Random number in API URL

Raghav-Modani opened the pull request:
resolves #8 
Added the functionality to call the API endpoint from some place passing the random number in the URL and returning the JSON response containing the number passed. (Used dynamic routes)

RuanAragao commented:
At another time we will need dynamic routes, but to compare the property of a ticket.

Raghav-Modani commented:
Hey,@RuanAragao! I have removed the dynamic API routes and added the simple functionality to return the JSON containing 8-digit number from the API endpoint. Made some modifications to the random number generator function to return a 8-digit number and used it in the API endpoint.
